<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>227</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="host_label">
       <property name="text">
        <string>Хост</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="port_label">
       <property name="text">
        <string>Порт</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="login_label">
       <property name="text">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pass_login">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="host_input"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="port_input"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="login_input"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="pass_input"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="connect_button">
       <property name="text">
        <string>Соединиться</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="close_button">
       <property name="text">
        <string>Закрыть БД</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="pass_login_2">
       <property name="text">
        <string>БД</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="db_name_input"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="combo_table_box">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="get_table_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить таблицы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблицы БД</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sql_request_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите SQL запрос</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат запроса</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="sql_response_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="do_request_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить запрос</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_output">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="actions_combo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionitalic">
   <property name="text">
    <string>italic</string>
   </property>
  </action>
  <action name="actionbold">
   <property name="text">
    <string>bold</string>
   </property>
  </action>
  <action name="actionunderline">
   <property name="text">
    <string>underline</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
